Dnn.AdminExperience/Dnn.PersonaBar.Extensions/Services/ServerSettingsSmtpHostController.cs

```diff
@@ -1,190 +1,191 @@
+                    PortalController.UpdatePortalSetting(portalId, "SMTPConnectionLimit", request.SmtpConnectionLimit, false);
+                    PortalController.UpdatePortalSetting(portalId, "SMTPMaxIdleTime", request.SmtpMaxIdleTime, false);
+                    PortalController.UpdatePortalSetting(portalId, "SMTPAuthentication", request.SmtpAuthentication.ToString(), false);
+                    PortalController.UpdatePortalSetting(portalId, "SMTPUsername", request.SmtpUsername, false);
+                    PortalController.UpdateEncryptedString(portalId, "SMTPPassword", request.SmtpPassword, Config.GetDecryptionkey());
+                    PortalController.UpdatePortalSetting(portalId, "SMTPEnableSSL", request.EnableSmtpSsl ? "Y" : "N", false);
+                }
+
+                DataCache.ClearCache();
+                return this.Request.CreateResponse(HttpStatusCode.OK, new { success = true });
+            }
+            catch (Exception exc)
+            {
+                Logger.Error(exc);
+                return this.Request.CreateErrorResponse(HttpStatusCode.InternalServerError, exc);
+            }
+        }
+
+        /// POST: api/Servers/SendTestEmail
+        /// <summary>
+        /// Tests SMTP settings.
+        /// </summary>
+        /// <param name="request"></param>
+        /// <returns></returns>
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public HttpResponseMessage SendTestEmail(SendTestEmailRequest request)
+        {
+            try
+            {
+                var mailFrom = request.SmtpServerMode == "h" ? Host.HostEmail : this.PortalSettings.Email;
+                var mailTo = this.UserInfo.Email;
+
+                var errMessage = Mail.SendMail(mailFrom,
+                    mailTo,
+                    "",
+                    "",
+                    MailPriority.Normal,
+                    Localization.GetSystemMessage(this.PortalSettings, "EMAIL_SMTP_TEST_SUBJECT"),
+                    MailFormat.Text,
+                    Encoding.UTF8,
+                    "",
+                    "",
+                    request.SmtpServer,
+                    request.SmtpAuthentication.ToString(),
+                    request.SmtpUsername,
+                    request.SmtpPassword,
+                    request.EnableSmtpSsl);
+
+                var success = string.IsNullOrEmpty(errMessage);
+                return this.Request.CreateResponse(success ? HttpStatusCode.OK : HttpStatusCode.BadRequest, new
+                {
+                    success,
+                    errMessage,
+                    confirmationMessage =
+                        string.Format(
+                            Localization.GetString(
+                                "EmailSentMessage",
+                                Components.Constants.ServersResourcersPath),
+                            mailFrom,
+                            mailTo),

```

Comment:
@thabaum did you intend to remove the `errorMessageSendingTestEmail` message?
```suggestion
                    confirmationMessage =
                        success
                            ? string.Format(
                                Localization.GetString(
                                    "EmailSentMessage",
                                    Components.Constants.ServersResourcersPath),
                                mailFrom,
                                mailTo)
                            : Localization.GetString("errorMessageSendingTestEmail"),
```